### ISSUE TITLE
Gotify notifications for users

### ISSUE THREAD

**Issue by mrmuiz** [Fallenbagel/jellyseerr#1056]
### Description

Now  users have only Pushbullit and Pushover. In Emby radarr and Sonarr i can give each user a own Gotify. But only one user cvan get jellyseerr notifications. 
![{9FAFB0EC-AA52-43A1-9DCD-38975EC959F2}](https://github.com/user-attachments/assets/d537bd34-4245-453d-ad5d-eaf32e333d2a)

### Desired Behavior

Gotify for each user. Or maybe even better this : 
![{DE3A9E0C-ADD0-4FBC-B8D9-925D05B84B92}](https://github.com/user-attachments/assets/b099983b-6e4b-4080-befa-e707767f71a6)
Also for users

### Code of Conduct

- [X] I agree to follow Jellyseerr's Code of Conduct

**Comment by Fallenbagel** [comment]
> Now users have only Pushbullit and Pushover

Please reword this as this is not true. Users get telegram, discord, pushbullet, Pushover and web push

**Comment by mrmuiz** [comment]
Not at my end?
![Screenshot_20241118_193135_Firefox](https://github.com/user-attachments/assets/e751d4e1-3050-4aa7-9d4f-961fcb5dc6d3)

**Comment by mrmuiz** [comment]
Strange 
![{93FE1DDD-AC7A-441A-8AC9-F59365343D92}](https://github.com/user-attachments/assets/6c727f03-29c1-4024-a5a2-b7797fbb3f22)

**Comment by Fallenbagel** [comment]
As mentioned in my first comment. I mentioned to reword the part that says only Pushover and pushbullet being the agents for users when that's not true as other types of agents exist (telegram, discord, Pushover, pushbullet, smtp, web push).

Seperate gotify for the user doesn't exist. So this feature request is valid. But the description was wrong. I'm asking to remove it as to not confuse others who read this.

**Comment by mrmuiz** [comment]
> As mentioned in my first comment. I mentioned to reword the part that says only Pushover and pushbullet being the agents for users when that's not true as other types of agents exist (telegram, discord, Pushover, pushbullet, smtp, web push).
> 
> Seperate gotify for the user doesn't exist. So this feature request is valid. But the description was wrong. I'm asking to remove it as to not confuse others who read this.

I w ill remove. But  it does exist. It works perfectly in other  apps like radarr  sonarr emby etc.

**Comment by Fallenbagel** [comment]
> > As mentioned in my first comment. I mentioned to reword the part that says only Pushover and pushbullet being the agents for users when that's not true as other types of agents exist (telegram, discord, Pushover, pushbullet, smtp, web push).
> > 
> > Seperate gotify for the user doesn't exist. So this feature request is valid. But the description was wrong. I'm asking to remove it as to not confuse others who read this.
> 
> I w ill remove. But  it does exist. It works perfectly in other  apps like radarr  sonarr emby etc.
> 

I don't think you're understanding me at all. Gotify USER SETTINGS don't exist IN JELLYSEERR. So this feature request is VALID. But the part that says only Pushover & pushullet are the only agents EXPOSED TO USER IN JELLYSEERR is wrong. I'm not talking about gotify users not existing nor am I talking about other apps like radarr sonarr emby. I'm talking about jellyseerr.

**Comment by mrmuiz** [comment]
> I don't think you understood me. Gotify **USER SETTINGS** don't exist **in JELLYSEERR**. So this feature request is **VALID**.
> 
> But the part that says only Pushover & pushullet are the only agents **exposed to the users in jellyseerr** is wrong which you had just rectified.
> 
> > But  it does exist. It works perfectly in other  apps like radarr  sonarr emby etc.
> 
> I'm not talking about gotify users not existing nor am I talking about other apps like radarr sonarr emby. I'm talking about **jellyseerr user settings not having gotify as an option.**

I am sorry  for the  misunderstanding . I edited it.  I hope its more corect now.

**Comment by woutervanelten** [comment]
Please add
